install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -30,6 +30,73 @@ $ openstack user create --password *<password> <username>*
 $ openstack role add --user <username> --project <project> _member_
 ----
 
+The default quotas assigned to new RH OSP projects are not enough for
+{product-title} installations. It is recommended that you have a minimum of 30
+security groups, 200 security group rules, and 200 ports available in your
+quota. And if kuryr SDN is enabled, and specially if namespace isolation is
+used, it is recommended to further increase those quotas to at least 100
+security groups, 500 security group rules, and 500 ports.
+----
+$ openstack quota set --secgroups 100 --secgroup-rules 500 --ports 500 *<project>*
```

Comment:
Should secgroups not be 30 here? 
Why are the text and the value in the command different (or why is that not explained to the reader)?

Reply:
This was an example, and I simply got the number that are specified for kuryr SDN

Reply:
We should match the numbers with the text or explain why the example sets the limits.

Reply:
It is matching the numbers of the text, but I'll modify so that it matches the values for openshift-sdn instead of the ones for kuryr-sdn